ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -311,7 +311,9 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar, cl
         <GenericModal
           open={!!profileForModal}
           Content={
+            <>
```

Comment:
Resolved, removed ✅
The only reason I added a fragment because I was thinking to add another parent for cross button but later I did it with second approach.